### PR TITLE
Bug/vgnwb 226

### PR DESCRIPTION
**Pull request by tjquinn1**
fixed scroll issue

### CODE REVIEW

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by tjquinn1** [comment]
What conflicts are you getting? This runs fine on my machine

File: annotations/views/repository_views.py [hunk 1]
```diff
@@ -368,6 +368,7 @@ def repository_text_content(request, repository_id, text_id, content_id):
 
 @login_required
 def repository_text_add_to_project(request, repository_id, text_id, project_id):
+    print "yes"
```
Comment (prnvshrn) [inline annotations/views/repository_views.py]: Please remove this print statement

File: annotations/templates/annotations/project_details.html [hunk 2]
```diff
@@ -42,7 +42,7 @@
         <thead>
             <th class="col-xs-9">
                 <a class="h4" style="text-decoration: none; vertical-align: middle;" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}" id="order_title">Title</a>
-                <a class="btn btn-sm btn-default" aria-label="Left Align" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}">
+                <a class="btn btn-sm btn-default" aria-label="Left Align" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}">\
```
Comment (prnvshrn) [inline annotations/templates/annotations/project_details.html]: The \ symbol is after the anchor tag ends. Can you remove it or put it inside the anchor tag ?

File: annotations/templates/annotations/project_details.html [hunk 3]
```diff
@@ -54,6 +54,7 @@
             </th>
         </thead>
         {% for text in texts %}
+        <h6>{{ text }}</h6>
```
Comment (jdamerow) [inline annotations/templates/annotations/project_details.html]: this does not seem to be related to the ticket

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Review by hashknot — APPROVED** [review]

File: annotations/static/annotations/css/annotators/text.css [hunk 4]
```diff
@@ -25,6 +25,9 @@
 
 .relation-list {
     font-size: 0.8em;
+    height: 90vh;
```
Comment (jdamerow) [inline annotations/static/annotations/css/annotators/text.css]: Ideally this should take up all the space there is, not a defined height. also there is still an issue with the footer:
image

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: annotations/static/annotations/css/annotators/text.css (hunk 4 above)
Comment (tjquinn1) [reply annotations/static/annotations/css/annotators/text.css]: @jdamerow  I am having trouble figuring out what is wrong with this fix. I tried it on the display size for a 13in screen, ipad and ipad pro screens and all work as intended. Also vh is a percentage of the users viewport. So this element should take up 90% percent of the screen. If you want it to take up 100% of the screen.The affix property of the list/search element will have to be modified to "turn off" when a certain point is reached so that the whole thing scrolls up a little when the footer comes up otherwise the element will always stay fixed and will be in the way. This won't work on sizes smaller than an Ipad but the site is not made for mobile anyway


screen shot 2018-01-11 at 10 26 30 am
# Ipad
screen shot 2018-01-11 at 10 20 20 am

# Ipad Pro
screen shot 2018-01-11 at 10 19 46 am

# 13in Screen
screen shot 2018-01-11 at 10 19 09 am

Comment (jdamerow) [reply annotations/static/annotations/css/annotators/text.css]: hm, ja, not sure how I got the problem. Maybe I confused app versions. Let' s leave it the way it is. If it does turn out to have issues, we can always come back to it. Please reopen the PR.

**Review by jdamerow — APPROVED** [review]
